package.json

```diff
@@ -28,7 +28,7 @@
     "lodash": "^4.17.4",
     "lodash.isequal": "^4.5.0",
     "ripple-address-codec": "^4.1.1",
-    "ripple-binary-codec": "^0.2.7",
+    "ripple-binary-codec": "1.1.0-beta.0",
```

Comment:
We'll wanna remember to change this to 1.1 before merging.

Reply:
Thanks for the review! Given the Thanksgiving break, let's plan on merging and releasing the two libraries early next week ⭐